Adyen/Core/Core Protocols/DeviceDependant.swift

```diff
@@ -6,11 +6,11 @@
 
 import Foundation
 
-/// Represents any structure/object whose behaviour is device dependant.
+/// Represents any structure/object whose behavior is device dependent.
 /// :nodoc:
-public protocol DeviceDependant {
+public protocol DeviceDependent {
```

Comment:
Can this be considered a breaking change? I see it's used within the `ComponentManager`, so is this something to be change to `internal` level?

Anyway, if we move on with the rename, we should rename the file too.

Reply:
yes, indeed. on the other hand its just public because its in the `Adyen` module, but used in another module, and its a `:nodoc:`, so I think its okay, just thinking that no merchant should be using this.

I renamed the file :-)